pkg/test/framework/runtime/components/pilot/native.go

```diff
@@ -132,6 +155,7 @@ func (c *nativeComponent) Close() (err error) {
 	if c.stopChan != nil {
 		c.stopChan <- struct{}{}
 	}
+	galley.CloseGalleyServer(c.galleyServer)
```

Comment:
This doesn't look right to me. Why are we closing galley in Pilot component? Galley's lifecycle should be managed through the Galley test component